tests/integration/components/stepper-test.js

```diff
@@ -6,6 +6,16 @@ import { hbs } from 'ember-cli-htmlbars';
 module('Integration | Component | stepper', function (hooks) {
   setupRenderingTest(hooks);
 
+  hooks.beforeEach(function () {
+    this.owner.register(
+      'service:login',
+      class MockLoginService {
+        isLoggedIn = false;
+      }
+    );
+    this.login = this.owner.lookup('service:login');
+  });
```

Comment:
Can you explain what you trying to do here

Reply:
Removed